src/dbg/analysis/recursiveanalysis.cpp

```diff
@@ -39,7 +40,8 @@ void RecursiveAnalysis::SetMarkers()
         // This algorithm orders basic blocks and then iterates, growing the chunk downwards
         // Function ranges are collected in another ordered map for loop insertion
         const auto & function = functionItr.second;
-        std::map<Range, const BridgeCFNode*, RangeCompare> blockRanges, functionRanges;

```

Comment:
This has to be an `std::map` because of the way the `Range` works. It allows a lookup of a value _inside_ the range vs the exact range.

Reply:
thanks!